ploneorg/core/browser/foundationsponsor.pt

```diff
@@ -9,49 +9,192 @@
+                ${context/payment_frequency}<br />
+                ${context/payment_method}<br />
+                ${context/payment_amount} ${context/payment_currency}<br />
+                ${context/payment_date}
+              </td>
+            </tr>
+            <tr class="even">
+              <th>Start date</th>
+              <td>${context/start_date}</td>
+            </tr>
+            <tr class="odd">
+              <th>End date</th>
+              <td>${context/end_date}</td>
+            </tr>
+            <tr class="even">
+              <th>Last verified</th>
+              <td>${context/last_verified_date}</td>
+            </tr>
+            <tr class="odd">
+              <th>Expiration</th>
+              <td>${context/expires}</td>
+            </tr>
+            <tr class="even">
+              <th>Private notes</th>
+              <td tal:define="notes context/notes"
+                  tal:condition="notes"
+                  tal:content="structure notes/output"
+                  style="width: 250px">
+              </td>
+            </tr>
+            </tal:private>
+          </tbody>
+        </table>
+        <tal:hide condition="nothing">
+            <div class="sponsorship_type" tal:define="sponsorship_type context/sponsorship_type">
+                <h4>
+                    <span tal:content="python: sponsorship_type.title()">sponsorship_type</span> sponsor
+                </h4>
+            </div>
+        
+            <div class="website" tal:define="url context/website" tal:condition="url">
+              <h4 tal:content="structure python: '<a href=''%s''>%s</a>' % (url, url)">
+                web site URL
+              </h4>
+            </div>
+        
+            <div class="providerlisting" tal:define="url context/provider_listing_url" tal:condition="url">
+              <h4 tal:content="structure python: '<a href=''%s''>%s</a>' % (url, url)">
+                Plone.com provider listing
+              </h4>
+            </div>
+        
+            <div class="twitter" tal:condition="python: context.twitter is not None and context.twitter.strip() != ''">
+                <div tal:define="twitter_username python: context.twitter.strip().lstrip('@'); url string: https://twitter.com/${twitter_username}" >
+                    <h4 tal:content="structure python: '<a href=''%s''>@%s</a>' % (url, twitter_username)">
+                        Twitter
+                    </h4>
+                </div>
+            </div>
+        
+            <div class="location" tal:condition="context/city">
+              <h4>
+                <span tal:content="context/city">City</span>
+                <tal:country condition="context/country">
+                  (<span tal:content="context/country">Country</span>)
+                </tal:country>
+              </h4>
+            </div>
+        </tal:hide>
+      </div>
     </div>
 
-    <div class="logo">
+    <div class="logo" tal:condition="nothing">
```

Comment:
Do not be afraid to remove code - we have Git. Or do you need it for debugging? then enabled it i.e. for "Manage Portal" permission.